sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -425,104 +201,73 @@ def _test_series_binary_operations(self, pyfunc, name, input_data=None):
-
-    def test_series_float_describe(self):
-        self._test_case(usecase_series_describe, 'series_describe')
-
-    def test_series_float_dropna(self):
-        self._test_case(usecase_series_dropna, 'series_dropna')
-
-    def test_series_float_fillna(self):
-        self._test_case(usecase_series_fillna, 'series_fillna')
-
-    def test_series_float_idxmax(self):
-        self._test_case(usecase_series_idxmax, 'series_idxmax')
-
-    def test_series_float_idxmin(self):
-        self._test_case(usecase_series_idxmin, 'series_idxmin')
-
-    def test_series_float_isna(self):
-        self._test_case(usecase_series_isna, 'series_isna')
-
-    def test_series_float_max(self):
-        self._test_case(usecase_series_max, 'series_max')
 
-    def test_series_float_mean(self):
-        self._test_case(usecase_series_quantile, 'series_mean')
+def test_gen(name, params):
+    func_name = 'func'
 
-    def test_series_float_median(self):
-        self._test_case(usecase_series_median, 'series_median')
+    func_text = f"""\
+def {func_name}(self):
+  self._test_case(usecase_gen('{name}({params})'), 'series_{name}')
+"""
 
-    def test_series_float_min(self):
-        self._test_case(usecase_series_min, 'series_min')
+    global_vars = {'usecase_gen': usecase_gen}
 
-    def test_series_float_nlargest(self):
-        self._test_case(usecase_series_nlargest, 'series_nlargest')
+    loc_vars = {}
+    exec(func_text, global_vars, loc_vars)
+    _gen_impl = loc_vars[func_name]
 
-    def test_series_float_nsmallest(self):
-        self._test_case(usecase_series_nsmallest, 'series_nsmallest')
+    return _gen_impl
 
-    def test_series_float_nunique(self):
-        self._test_case(usecase_series_nunique, 'series_nunique')
 
-    def test_series_float_pow(self):
-        self._test_series_binary_operations(usecase_series_pow, 'series_pow')
+def test_gen_two_par(name, params):
+    func_name = 'func'
 
-    def test_series_float_prod(self):
-        self._test_case(usecase_series_prod, 'series_prod')
+    func_text = f"""\
+def {func_name}(self):
+  self._test_series_binary_operations(usecase_gen_two_par('{name}', '{params}'), 'series_{name}')
+"""
 
-    def test_series_float_quantile(self):
-        self._test_case(usecase_series_quantile, 'series_quantile')
+    global_vars = {'usecase_gen_two_par': usecase_gen_two_par}
 
-    def test_series_float_shift(self):
-        self._test_case(usecase_series_shift, 'series_shift')
+    loc_vars = {}
+    exec(func_text, global_vars, loc_vars)
+    _gen_impl = loc_vars[func_name]
 
-    def test_series_float_sort_values(self):
-        self._test_case(usecase_series_sort_values, 'series_sort_values')
+    return _gen_impl
 
-    def test_series_float_std(self):
-        self._test_case(usecase_series_std, 'series_std')
 
-    def test_series_float_sum(self):
-        self._test_case(usecase_series_sum, 'series_sum')
+cases = {
+    'abs': [], 'argsort': [],
```

Comment:
I think that storing call arguments in list of parameters have no sense. I think it could be simple string. What do you think? Does a list have advantages?